templates/download/server/arm.html

```diff
@@ -15,7 +15,7 @@
             <h1>Ubuntu Server for ARM</h1>
             <p>Ubuntu 16.04 LTS includes support for the very latest ARM-based server systems powered by certified 64-bit processors.</p>
             <p>Develop and test using over 50,000 software packages and runtimes &mdash; including Go, Java, Javascript, PHP, Python and Ruby &mdash; and deploy at scale using our complete scale-out management suite including MAAS and Juju. Ubuntu delivers server-grade performance on ARM, while fully retaining the reliable and familiar Ubuntu experience.</p>
-            <p><a class="external" href="http://www.ubuntu.com/certification/soc/">See all certified ARM systems</a></p>
+            <!-- p><a class="external" href="http://www.ubuntu.com/certification/soc/">See all certified ARM systems</a></p -->
```

Comment:
I agree in principle, but I think in this case it's okay 'cos it's going to be added back very soon (a week or so) and it's useful to remember that.